### PR TITLE
Match merging: Use default values for CLI 

### PR DESCRIPTION
**Pull request by uuqjz**
Fixed an issue where the default values where not used for the CLI.

### CODE REVIEW

**Comment by uuqjz** [comment]
@tsaglam Ready for Review!

File: cli/src/test/java/de/jplag/cli/MergingOptionsTest.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
+package de.jplag.cli;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+import org.junit.jupiter.api.Test;
+
+import de.jplag.merging.MergingOptions;
+
+public class MergingOptionsTest extends CommandLineInterfaceTest {
```
Comment (tsaglam) [inline cli/src/test/java/de/jplag/cli/MergingOptionsTest.java:10]: Add a short JDoc comment here. Along the lines of "Test cases for the options of the match merging mechanism."

File: cli/src/test/java/de/jplag/cli/MergingOptionsTest.java [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    @Test
```
Comment (tsaglam) [inline cli/src/test/java/de/jplag/cli/MergingOptionsTest.java:11]: Add @Description("...") here. This is what is shown in your IDE. Can be a brief sentence describing the tested behavior (test if default values are used when cresting merging options).

**Review by tsaglam — CHANGES_REQUESTED** [review]
Just two minor things. Not all existing test cases fulfill this, but new ones should.

**Review by tsaglam — APPROVED** [review]

**Pull request merged**